doc/whats_new/v1.1.rst

```diff
@@ -595,6 +595,25 @@ Changelog
   in the multiclass case when ``multiclass='ovr'`` which will return the score
   per class. :pr:`19158` by :user:`Nicki Skafte <SkafteNicki>`.
 
+- |Fix| Fixed a bug in :func:`metrics.normalized_mutual_info_score` which could return
+  unbounded values. :pr:`22635` by :user:`Jérémie du Boisberranger <jeremiedbb>`.
+
+:mod:`sklearn.manifold`
+.......................
+
+- |Enhancement| :func:`manifold.spectral_embedding` and
```

Comment:
The manifold section was moved up. If you used git merge, then this diff could be related to https://github.com/scikit-learn/scikit-learn/issues/21516

Reply:
fixed, thanks